@here/harp-geoutils/lib/projection/EarthConstants.ts

```diff
@@ -10,4 +10,16 @@ export class EarthConstants {
 
     /** The equatorial radius in meters. */
     static EQUATORIAL_RADIUS: number = 6378137.0;
+
+    /**
+     * Lowest point on dry land: Shore of the Dead Sea.
+     * https://en.wikipedia.org/wiki/Extreme_points_of_Earth
+     */
+    static MIN_ELEVATION: number = -432.65;
+
+    /**
+     * Mount Everest.
+     * https://en.wikipedia.org/wiki/Extreme_points_of_Earth
+     */
+    static MAX_ELEVATION: number = 8.848;
```

Comment:
I think you don't mean to have the . there ?

Reply:
good catch